src/components/HearingImageList.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import HearingImage from './HearingImage';
+
+class HearingImageList extends React.Component {
+  render() {
+    const {images} = this.props;
+    return (<div>
+      {images.map((image) => <HearingImage data={image}/>)}
```

Comment:
[Arrays of components must have `key` props.](http://facebook.github.io/react/docs/multiple-components.html#dynamic-children)